pkgs/top-level/all-packages.nix

```
@@ -11526,9 +11526,13 @@ with pkgs;
 
   qovery-cli = callPackage ../tools/admin/qovery-cli { };
 
-  qownnotes = darwin.apple_sdk_11_0.callPackage ../applications/office/qownnotes {
+  qownnotes-qt5 = if stdenv.isDarwin then darwin.apple_sdk_11_0.callPackage ../applications/office/qownnotes {
```

Renaming qownnotes to qownnotes-qt5 is unnecessary. Keeping the original name is fine, otherwise you would need to add aliases (`pkgs/top-level/aliases.nix`).

---

@totoroot, @wineee, just for my understanding, is there an advantage in keeping both the qt5 and qt6 versions instead of just switching to qt6?

---

```suggestion
  qownnotes = darwin.apple_sdk_11_0.callPackage ../applications/office/qownnotes {
```
for linux that wrapper is a noop